recipes/autogenes/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set version = "1.0.4" %}
+
+package:
+  name: autogenes
+  version: {{ version }}
+
+source:
+  url:
+    - https://files.pythonhosted.org/packages/f0/04/4768c40962c38dd15b6cbd67d23f1c0d97d2dedeaa6558f6f8689458b4c5/autogenes-{{ version }}.tar.gz
+  sha256: 8daa6e9aa97c7ddf45ab0906307a4875a25bd99301be0838bb17e313d837446a
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv
+  noarch: generic
+
+requirements:
+  host:
+    - python >=3.6
+    - pandas >=0.25.1
+    - anndata >=0.6.22.post1
+    - numpy >=1.17.2
+    - dill >=0.3.1.1
+    - deap >=1.3.0
+    - scipy >=1.3
+    - cachetools >=3.1.1
+    - scikit-learn >=0.21.3
+    - matplotlib >=3.0
```

Comment:
`noarch: python` packages don't require all dependencies under `host`. It's usually just `python` and `pip`. 

If a package has some `C` dependencies, you would additionally require a compiler. 

Explanation: `host` are the packages needed to *build/install* the package, `run` are the dependencies required to *run/execute* the package.